db/schema.rb

```diff
@@ -26,6 +26,15 @@
   add_index "application_letters", ["event_id"], name: "index_application_letters_on_event_id"
   add_index "application_letters", ["user_id"], name: "index_application_letters_on_user_id"
 
+  create_table "application_notes", force: :cascade do |t|
```

Comment:
Why do you have changes in this file when you didn't create any migrations? Have a loot at [this page](http://edgeguides.rubyonrails.org/active_record_migrations.html#schema-dumping-and-you) to get a better understanding of schema files if you didn't just forget to commit the migration file 😀